### PR TITLE
various formatting tweaks

### PR DESCRIPTION
**Pull request by afeld**
* Fix links
* Remove hard line wrapping in AWS file
* Fix some spelling
* Fix/improve Markdown formatting

/cc @leahbannon

### CODE REVIEW

**Comment by afeld** [comment]
I should also note that the hard line wrapping was being interpreted as such by Jekyll/Kramdown. This was unexpected, but gives me a good excuse to switch to soft line wrapping, which I prefer anyway :smirk: 

Before:

screen shot 2015-12-21 at 10 46 59 am

After:

screen shot 2015-12-21 at 10 47 25 am

**Comment by afeld** [comment]
Ah, apparently this is configurable in Kramdown...see the `hard_wrap` option. _Meh._

http://kramdown.gettalong.org/rdoc/Kramdown/Options.html

File: pages/aws.md [hunk 1]
```diff
@@ -38,20 +22,11 @@ Your username will be created for you during onboarding. Check the MFA Token box
 
 ### Servers
 
-The core service of AWS is the [Elastic Compute Cloud](http://aws.amazon.com/documentation/ec2/) (EC2).
-These are virtual machines just like on your computer, but hosted in the AWS
-environment. You receive access through a GUI, the [AWS Management Console](https://18f.signin.aws.com),
-an [API](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/query-apis.html),
-and to the guest [OS through SSH](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html).
+The core service of AWS is the [Elastic Compute Cloud](http://aws.amazon.com/documentation/ec2/) (EC2). These are virtual machines just like on your computer, but hosted in the AWS environment. You receive access through a GUI, the [AWS Management Console](https://18f.signin.aws.com), an [API](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/query-apis.html), and to the guest [OS through SSH](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html).
```
Comment (DavidEBest) [inline pages/aws.md]: https://18f.signin.aws.com doesn't resolve. Should this be https://18f.signin.aws.amazon.com/console instead?

File: pages/ato-and-you.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
 ---
 title: Your ATO
-permalink: /ato/
+permalink: /ato-and-you/
 ---
 
-An Authority to Operate (ATO) is a complicated security review process that is required for deploying any web service on the public web. [You can learn more about ATOs here](https://pages.18f.gov/before-you-ship/ato/)
+An Authority to Operate (ATO) is a complicated security review process that is required for deploying any web service on the public web. [Learn more about ATOs](ato/).
```
Comment (DavidEBest) [inline pages/ato-and-you.md]: When I run this locally, this attempts to open http://localhost:4000/ato-and-you/ato/, which is not found.

File: pages/ato-and-you.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
 
 First, make sure that your project is not more complicated than the typical Open Data ATO. The following are signs that your ATO process will be more complicated and you should consult Noah when you begin planning:
-* Anytime you touch Sensitive PII (see explanation: https://hub.18f.gov/standards/aws/)
+
+* Are you dealing with Sensitive Personally Identifiable Information (PII)? ([see explanation](../infrastructure/aws/))
```
Comment (DavidEBest) [inline pages/ato-and-you.md]: Seems odd to redirect to `aws` for PII. Should PII be pulled into it's own page? If not, could we put an anchor on it and link directly to that section?

File: pages/aws.md (hunk 1 above)
Comment (afeld) [inline pages/aws.md]: Yeah, was worrying about broken links in #39.

File: pages/ato-and-you.md (hunk 3 above)
Comment (afeld) [inline pages/ato-and-you.md]: Good call – will tackle that separately. #45

**Comment by afeld** [comment]
Updated!

**Comment by leahbannon** [comment]
LGTM :+1: Should I merge?

**Comment by DavidEBest** [comment]
I ran locally, and it looks good. :+1:

**Comment by afeld** [comment]
If pull requests look good, feel free to merge!

**Pull request merged**